templates/article.html

```diff
@@ -23,6 +23,10 @@
 {{ smo_metadata(article) }}
 {% endblock meta_tags_in_head %}
 
+{% block head_links %}
```

Comment:
add

```
{ super() }}
```

Immediately after opening the block.

For example https://github.com/Pelican-Elegant/elegant/blob/master/templates/article.html#L17

Reply:
done!, thx